source/templates.tex

```diff
@@ -4747,7 +4747,7 @@
 };
 
 template<class T, template<class> class U = T::template Base> struct Third { };
-Third<Derived<int> > t;                 // OK: default argument uses injected-class-name as a template
+Third<Derived<int>> t;                  // OK: default argument uses injected-class-name as a template
```

Comment:
I think we don't want to change this under the "core sections show a variety of syntactically valid styles" doctrine.

Reply:
@jensmaurer Could we at least make the spacing symmetric? :)

Reply:
Ah, the one place I didn't look when checking for more instances was... [temp]. I can still wait for @zygoloid to look this over if we want, but I agree with you that we shouldn't make the change